Directory.Build.props

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+
+  <PropertyGroup>
+    <Configurations>Debug;Release;Debug_ReadOnly;Release_ReadOnly</Configurations>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)\cecil.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Copyright>Copyright © 2008 - 2015 Jb Evain</Copyright>
+    <AssemblyVersion>0.10.0.0</AssemblyVersion>
+    <FileVersion>0.10.0.0</FileVersion>
+    <InformationalVersion>0.10.0.0-beta6</InformationalVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="$(Configuration.StartsWith('Debug'))">
+    <DebugType>full</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="$(Configuration.StartsWith('Release'))">
+    <DebugType>pdbonly</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+  </PropertyGroup>
```

Comment:
i think the two property groups are redundant? the defaults for release and debug should suffice

Reply:
The defaults use the new PDB format which causes a variety of errors. These conditions force the use of the old PDB format, at least when building on Windows.

Reply:
What are the variety of issues with the new PDB format?

Reply:
Very old NUnit 2 test adapter uses very old cecil, resulting in total failure to support test runs.

Reply:
:)

We could look into using NUnit 3.

Reply:
> We could look into using NUnit 3.

I haven't looked into the viability of this. I do know that OpenCover doesn't support portable PDB, so I normally keep these two property groups from the start when converting a project to the new project system.

Reply:
ilspy also has issues. basically any tool that integrates the pdb need to be updated. so we r in a period of transition